app/build.gradle

```diff
@@ -125,6 +128,11 @@ configurations {
     geckoImplementation {}
 }
 
+// Generate markdown docs for the collected metrics.
+ext.gleanGenerateMarkdownDocs = true
+ext.gleanDocsDirectory = "$rootDir/docs/glean"
+apply from: 'https://github.com/mozilla-mobile/android-components/raw/v' + rootProject.ext.moz_components_version + '/components/service/glean/scripts/sdk_generator.gradle'
```

Comment:
This is no longer the supported way to use the Glean code generation plugin.  See here: 

https://mozilla.github.io/glean/book/user/adding-glean-to-your-project.html#adding-new-metrics

Reply:
@mdboom this is using legacy a-c/glean version (24.0.0), I get the following compilation error:
`Plugin with id 'org.mozilla.telemetry.glean-gradle-plugin' not found.`

Reply:
Ok.  Be aware that the `sdk_generator.gradle` script has no way to ensure that it's running the correct version of the Glean generator, so this problem of generating the wrong code could re-occur in the future.  If there's any way you could upgrade to a-c 27.0.0 or later, you would have access to the Glean Gradle plugin, which resolves that issue.

Reply:
@mdboom I think we will address this in https://github.com/mozilla-mobile/firefox-tv/issues/2990.
AFAIK, we are currently lacking QA support for big a-c upgrade. I think it's safer to upgrade a-c version to 24 for `deletion-request ping` support.

Reply:
@mcomella what do you think about this ^?

Reply:
Since we're only expecting to build one build from this in the next few weeks, I'd think this is probably fine to move forward with this.

@mdboom Do you agree?

@psymoon If mdboom agrees, can you file an issue about "upgrade to a-c 27 and use glean plugin" and talk to athomasmoz to ensure it's at the top of the backlog? It should be the first thing we do before adding more code (or at least validate that it won't break). Ideally, we'll upgrade to the latest version, but if not, we should do that issue.

Reply:
That makes sense.  Thanks for checking.

Reply:
Filed https://github.com/mozilla-mobile/firefox-tv/issues/2993.